doc/tutorial/nan_tutorial.txt

```diff
@@ -0,0 +1,65 @@
+
+.. _nan_tutorial:
+
+=================
+Dealing with NaNs
+=================
+
+Having a model yielding NaNs or Infs is quite common if some of the tiny
+components in your model are not set properly. NaNs are hard to deal with
+because sometimes it is caused by a bug or error in the code, sometimes it's
+because of the numerical stability of your actual computing systerm, and even,
```

Comment:
Actually, I don't know what you mean by "computing system", I see you mention algorithm below.